### PR TITLE
Stop erroneously flagging variables called "version"

### PR DESCRIPTION
**Pull request by aednichols**
The simplest (probably not the best) fix for https://github.com/broadinstitute/winstanley/issues/91.

### CODE REVIEW

**Comment by cjllanwarne** [comment]
👍 - it's not quite as thorough as a new lexer state would be, but it's also a one-line change that can immediately stop us from annoying everyone... so works for me!

[Approved with PullApprove](https://pullapprove.com/broadinstitute/winstanley/pull-request/98/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/winstanley)

**Review by cjllanwarne — APPROVED** [review]

**Review by cjllanwarne — CHANGES_REQUESTED** [review]
Actually, can we have a new parser test for this, to make sure we don't regress?

File: tests/testData/parsing/variable_named_version_draft_2.wdl [hunk 1]
```diff
@@ -0,0 +1,3 @@
+workflow a {
+  String version = "a"
+}
```
Comment (cjllanwarne) [inline tests/testData/parsing/variable_named_version_draft_2.wdl]: https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file

**Review by cjllanwarne — APPROVED** [review]

File: tests/testData/parsing/variable_named_version_draft_2.wdl (hunk 1 above)
Comment (aednichols) [reply tests/testData/parsing/variable_named_version_draft_2.wdl]: Ah, I inadvertently left this setting behind when I switched from Ultimate to Community - thanks

**Pull request merged**